src/options.rs

```diff
@@ -493,6 +450,31 @@ impl Unit {
             Hour | Minute | Second | Millisecond | Microsecond | Nanosecond
         )
     }
+
+    /// `13.19 LargerOfTwoTemporalUnits ( u1, u2 )`
+    ///
+    /// Spec: <https://tc39.es/proposal-temporal/#sec-temporal-largeroftwotemporalunits>
+    //
+    // Spec last accesed: 2025-05-16, <https://github.com/tc39/proposal-temporal/tree/c150e7135c56afc9114032e93b53ac49f980d254>
+    #[inline]
+    #[must_use]
+    pub fn larger(u1: Unit, u2: Unit) -> Unit {
+        // 1. For each row of Table 21, except the header row, in table order, do
+        //     a. Let unit be the value in the "Value" column of the row.
+        for unit in UNIT_VALUE_TABLE {
+            // b. If u1 is unit, return unit.
+            if u1 == unit {
+                return unit;
+            }
+            // c. If u2 is unit, return unit.
+            if u2 == unit {
+                return unit;
+            }
+        }
+
+        // TODO: Is this correct???
+        Unit::Auto
```

Comment:
I'm not sure if this is correct? the spec has not auto in the [`Table 21`](https://tc39.es/proposal-temporal/#table-temporal-units) so what is returned here?

```
For each row of Table 21, except the header row, in table order, do
       a. Let unit be the value in the "Value" column of the row.
       b. If u1 is unit, return unit.
       c. If u2 is unit, return unit.
```